wordle/readme.md

```diff
@@ -0,0 +1,33 @@
+# Wordle CLI Workshop
+
+This is a workshop where we, as a class, build a Wordle CLI clone in Node JS.
+
+## Learning Objectives
+
+```objectives
+- [ ] Give trainees the experience of building a fully playable game from scratch, starting with nothing but an idea
+- [ ] Give trainees some understanding of how to find and use NPM dependencies in a project
+- [ ] Walk trainees through the process of breaking down a high-level idea into smaller, less intimidating bits of functionality
+- [ ] Inspire trainees to take their own ideas and build something cool with whatever skills they have
+```
+
+## Requirements
+
+- Split the cohort into groups of 3 to 4 people
+- Introduction slides: https://barath-cyf.netlify.app/wordle/
```

Comment:
```suggestion
```

Reply:
From playing the game itself on NY Times they'll build sufficient intuition to tackle this problem and the slides won't add anything that they can't learn from playing the game itself.